targets/TARGET_WIZNET/TARGET_W7500x/analogin_api.c

```diff
@@ -58,11 +58,11 @@ void analogin_init(analogin_t *obj, PinName pin)
     obj->pin = pin;
 
     // The ADC initialization is done once
-    if (adc_inited == 0) {
+    //if (adc_inited == 0) {
```

Comment:
Problems occurred when several Two or more pins were used...
So we modified this code.